CMakeLists.txt

```diff
@@ -553,7 +553,7 @@ SET(DATADIR "" CACHE STRING "Default vegastrike data directory")
 
 # Hide our temporary variable and initialize it to empty
 SET(CPU_OPTS  )
-SET(DEFINES " -DNV_CUBE_MAP -DBOOST_PYTHON_NO_PY_SIGNATURES ")
+SET(DEFINES " -D NV_CUBE_MAP=1 -D BOOST_PYTHON_NO_PY_SIGNATURES=1 ")
```

Comment:
It would be good if the values specified mapped to some configuration variables elsewhere to make them easy to change and document. $0.02 FWIW

Reply:
I am now working on CMake stuff in a separate branch, although I'm having a LOT of issues building the project, on both Linux and Windows.

Reply:
@stephengtuggy cool - let me know if you need help. I haven't successfully built it yet myself, but am certainly willing to help out where I can.